src/layers/src/Pane.js

```diff
@@ -32,90 +79,26 @@ const Pane = memo(
     } = props
     const theme = useTheme()
 
-    function getHoverElevationStyle(hoverElevation, css) {
-      if (!Number.isInteger(hoverElevation)) return {}
-
-      return {
-        transitionDuration: '150ms',
-        transitionProperty: 'box-shadow, transform',
-        transitionTimingFunction: `cubic-bezier(0.0, 0.0, 0.2, 1)`,
-        ':hover': {
-          ...(css[':hover'] || {}),
-          transform: 'translateY(-2px)',
-          boxShadow: theme.elevations[hoverElevation]
-        }
-      }
+    const styles = {
+      ...css,
+      background: theme.colors.background[background] || background,
+      boxShadow: theme.elevations[elevation],
```

Comment:
I was going to leave a comment if it makes sense to namespace elevations under `tokens`, but I think once we go through and do all the theme cleanup of olde, we'll be left with `theme.tokens` housing everything, aka we should probably just call everything in there `theme`. Don't mind me!